rotkehlchen/data/importer.py

```diff
@@ -875,11 +882,17 @@ def _consume_nexo(self, csv_row: Dict[str, Any]) -> None:
             log.debug(f'Ignoring rejected nexo entry {csv_row}')
             return
 
-        asset = symbol_to_asset_or_token(csv_row['Currency'])
+        asset = asset_from_nexo(csv_row['Currency'])
         amount = deserialize_asset_amount_force_positive(csv_row['Amount'])
         entry_type = csv_row['Type']
         transaction = csv_row['Transaction']
 
+        if entry_type == 'Exchange':
+            self.db.msg_aggregator.add_warning(
```

Comment:
So this happens every time you have an exchange entry? Would it not spam the user? Even though I guess no time is here so all similar messages would be hidden.

But why ignore? What should the user do? What can they do?

Reply:
The CSV is missing basic information to create the trade. I don't have a way to check this but I guess that from the UI the user could find the exchange information and manually add it to rotki. As you said since we ignore every entry with the same message the frontend should group the messages.

The csv provides the output amount but no input amount is provided nor the rate of the trade